packages/shared/src/modules/user-follows/libs/enums/user-follows-api-path.enum.ts

```diff
@@ -0,0 +1,6 @@
+const UserFollowsApiPath = {
+	FOLLOW: "/:userId/followers",
+	UNFOLLOW: "/:userId/followers/:id",
```

Comment:
for the api paths, we will name them not by meaning, but by actual path, here is the pattern
```suggestion
	$USER_ID_FOLLOWERS: "/:userId/followers",
	$USER_ID_FOLLOWERS_$ID: "/:userId/followers/:id",
```